client/src/components/SearchInput.js

```diff
@@ -26,7 +28,11 @@ export default function SearchInput({ onChange, size = 'medium' }) {
   }
 
   // this is just the help make the select look more like the designs
-  const selectWidth = resourceType === 'ALL' ? '60px' : '156px'
```

Comment:
Should the comment above this have been moved as well?

Reply:
yup 👍